bin/add-feed.sh

```diff
@@ -0,0 +1,45 @@
+podcast_id=$1
+base_directory="../feeds"
+
+cd "$base_directory" || exit
+
+if [ ! -d "$podcast_id" ]; then
+    mkdir "$podcast_id"
+fi
```

Comment:
But what if folder already exists?

Reply:
If the folder already exists, the script will simply skip the creation step and proceed with the rest of the operations. The if condition checks for the existence of the directory using `[ ! -d "$PODCAST_ID" ]`.

Reply:
No. It should stop and show warning about feed existence.